### PR TITLE
trpc versions from alpha to beta 📦

### PR DESCRIPTION
**Pull request by azezsan**
I updated trpc versions to the beta versions and modified trpc utils files in both Nextjs and Expo to be compatible according to trpc's 10.x docs

### CODE REVIEW

**Comment by azezsan** [comment]
I tested on: 
PC for nextjs and expo with android emulator 
Linux wsl for nextjs only
both worked with no problems

File: apps/nextjs/src/utils/trpc.ts [hunk 1]
```diff
@@ -10,12 +11,16 @@ const getBaseUrl = () => {
...
+      links: [
+        httpBatchLink({
+          url: `${getBaseUrl()}/api/trpc`,
+        }),
+      ],
     };
   },
-  ssr: false,
-});
+  ssr: true,
```
Comment (juliusmarminge) [inline apps/nextjs/src/utils/trpc.ts:25]:
```suggestion
  ssr: false,
```

**Review by juliusmarminge — COMMENTED** [review]
Thanks, i think you need to make some changes to the API package too though

**Review by juliusmarminge — APPROVED** [review]

**Comment by azezsan** [comment]
> Thanks, i think you need to make some changes to the API package too though.
> 
> EDIT: nvm the ones i thought of were already done

happy to help ^^

**Pull request merged**